exporter/awsxrayexporter/internal/translator/http.go

```diff
@@ -110,6 +110,9 @@ func makeHTTP(span ptrace.Span) (map[string]pcommon.Value, *awsxray.HTTPData) {
 		case string(conventions.URLPathKey):
 			urlParts[key] = value.Str()
 			hasHTTP = true
+		case string(conventions.URLQueryKey):
+			urlParts[key] = value.Str()
+			hasHTTP = true
```

Comment:
Maybe we just combine these two cases:
```suggestion
		case string(conventions.URLPathKey), string(conventions.URLQueryKey):
			urlParts[key] = value.Str()
			hasHTTP = true
```